Lib/test/test_support.py

```diff
@@ -1,3 +1,5 @@
+import contextlib
+import logging
```

Comment:
Oups, my bad, I forgot to apply the patch here. `import logging` should come just after `import io`.